dotCMS/src/main/java/com/dotmarketing/velocity/directive/ContentletDetail.java

```diff
@@ -54,13 +53,21 @@ else if(type.baseType() == BaseContentType.WIDGET  && !Config.getBooleanProperty
       throw new ResourceNotFoundException("cannnot find contentlet id " +  argument + " lang:" + params.language);
     }
 
+      // _show_working_ context variable is used on Container Services. If the time machine date is after the Publish
+      // date of the Contentlet (identifier data) we need to show the working. If not we only show the live.
+      //
+      // #if($UtilMethods.isSet($_ident.sysPublishDate) && $_tmdate.after($_ident.sysPublishDate))
+      //    #set($_show_working_=true)
+      //
+      boolean showWorking = false;
 
-    return  (params.live) 
-        ? "/live/"      + argument + "_" + cv.getLang() + "." + EXTENSION
-        : "/working/"   + argument + "_" + cv.getLang() + "." + EXTENSION;
-
-
+      if (context.get("_show_working_") != null && (boolean)context.get("_show_working_")) {
```

Comment:
I thought on that but at the end I did it in ContentletDetail.java because:
1. I don't have access to the Context in RenderParams.
2. Even if I have access to the Context I don't like the idea to keep inspecting the Context for a variable that we only use in ContentletDetail.java.